lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4404,6 +4404,9 @@ namespace {
     AccessorDecl *tryCreateConstexprAccessor(const clang::VarDecl *clangVar,
                                              VarDecl *swiftVar) {
       assert(clangVar->isConstexpr());
+      auto *Eval = clangVar->ensureEvaluatedStmt();
+      if (cast<clang::Expr>(Eval->Value)->isValueDependent())
+        return nullptr;
```

Comment:
This will drop the member on the floor. Is that the intended behavior?

Reply:
I'm not entirely sure, but I _think_ what's going on here is we aren't able to import this because it uses a dependent type, and we currently cannot import dependent types. If that's the case, I think it is OK to just drop this one the floor (especially given that all we're actually dropping on the floor is the constexpr accessor, right here at least), because we don't really have a way to import it yet. Once we support dependent types, then we should fix this so that it behaves properly (see my comment above).